runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/internal/runtime/InternalPlatform.java

```diff
@@ -391,7 +397,7 @@ public IPath getLocation() throws IllegalStateException {
 				if (explicitInitRequired) {
 					// See bug 514333: don't allow clients to initialize instance location if the
 					// instance area is not explicitly defined yet
-					throw new IllegalStateException(CommonMessages.meta_instanceDataUnspecified);
+					throw new IllegalStateException(""); //$NON-NLS-1$
```

Comment:
Why replace with empty (non externalized) string?

Reply:
that tooling...